<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{background-color:rgb(0, 170, 255)}</string>
   </property>
   <widget class="QPushButton" name="DesifrujButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:rgb(223, 148, 111)}</string>
    </property>
    <property name="text">
     <string>Dešifruj</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="matrix_browser">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="vstup_text_browser">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_edit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SifrujButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:rgb(223, 148, 111)}</string>
    </property>
    <property name="text">
     <string>Šifruj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="s_text_edit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="sifra_browser">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kluc1_edit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kluc2_edit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(255, 255, 255)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="matrix1_browser">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="d_text_browser">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>590</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZmazButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>310</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:rgb(223, 148, 111)}</string>
    </property>
    <property name="text">
     <string>Zmaž</string>
    </property>
   </widget>
   <widget class="QLabel" name="d_text_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dešifrovaný text :</string>
    </property>
   </widget>
   <widget class="QLabel" name="sifra_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Šifrovaný text :</string>
    </property>
   </widget>
   <widget class="QLabel" name="vstup_text_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vstupný text :</string>
    </property>
   </widget>
   <widget class="QLabel" name="s_text_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Šifrovaný text :</string>
    </property>
   </widget>
   <widget class="QLabel" name="kluc1_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Klúčové slovo :</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Text :</string>
    </property>
   </widget>
   <widget class="QLabel" name="kluc2_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Klúčové slovo :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string> Playfair cipher</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_pair_browser">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>220</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="sifra_pair_browser">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>570</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{background-color:rgb(255, 255, 255)}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Matica</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Text v pároch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>370</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Matica</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>550</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Šifrovaný text v pároch</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1239</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
